config/model_configs/les_isdac_box.yml

```diff
@@ -0,0 +1,32 @@
+# device: CPUSingleThreaded
+job_id: "les_isdac_box"
+initial_condition: "ISDAC" 
```

Comment:
The current assembly and yaml file information is reasonable, but let's start a separate issue to define `cases` that collect all relevant forcing terms for a given configuration.